src/index.html

```diff
@@ -9,6 +9,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="main-logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="main_logo">
+        </a>
+      </div>
+
+      <nav class="nav">
+
+        <ul class="nav_list">
+
+          <li class="nav_link_box">
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav_list">
          <li class="nav_link_box">
```

Reply:
there shouldn’t be empty line between child and parent element

Reply:
done